src/DynamoUtilities/DynamoUtilities.csproj

```diff
@@ -49,6 +49,7 @@
     <Reference Include="System" />
     <Reference Include="System.configuration" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Numerics" />
```

Comment:
Do we need this reference?

Reply:
Yes, `BigInteger` is defined in `System.Numerics`.